assets/map-layout.scss

```diff
@@ -0,0 +1,20 @@
+---
+---
+
+@import "{{ site.theme }}";
+
+
+
+
+
+
+.wrapper {
```

Comment:
The default header that we're using from Jekyll uses this class already; can you rename this so that it does not override that one?